S3.Rmd

```diff
@@ -498,7 +498,7 @@ There are two wrinkles to be aware of when you create a new method:
 *   First, you should only ever write a method if you own the generic or the
     class. R will allow you to define a method even if you don't, but it is
     exceedingly bad manners. Instead, work with the author of either the 
-    generic or the class to add the method in their code.
+    generic or the class to add the method in his or her code.
```

Comment:
I'm fine with singular they, so can you please revert?

Reply:
I also like it better. I think there are other instances of "his or her" in other chapters. I can do a global check after everything is merged so it's consistent throughout.